server/v2/models/User.js

```diff
@@ -0,0 +1,38 @@
+import Bcrypt from '../helpers/Bcrypt';
+import DBConnection from './DBConnection';
+import Queries from './Queries';
+
+class User {
+  static async getUserByEmail(email) {
+    await DBConnection.query(Queries.userTable.createTable);
+
+    const { rows } = await DBConnection.query(Queries.userTable.isUserExist, [email]);
+
+    return rows;
+  }
+
+  static async saveUser({
+    fname, lname, email, phone, username, password,
+  }) {
+    const encryptedPassword = Bcrypt.encryt(password);
+
+    const { rows } = await DBConnection.query(
+      Queries.userTable.insertUser,
+      [fname, lname, email, phone, username, encryptedPassword],
+    );
+    return rows[0];
+  }
+
+  static async checkaUser({ email, password }) {
```

Comment:
You should change the way you are naming your variables and functions. `checkUser` instead of ``checkaUser``

Reply:
Yes, let me change it